slides/workshopslides_gitDevOps.html

```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
+<ul>
 <li>Git records all the version control information within a hidden folder called .git</li>
 </ul>
+</div>
 </section>
 
 <section id="and-github-and-dev-ops" class="title-slide slide level1 center">
 <h1>And GitHub and Dev Ops…</h1>
+<div class="fragment fade-up">
 <p>GitHub and Dev Ops provide a space to store and interact with a repository.</p>
+</div>
+<div class="fragment fade-up">
 <ul>
 <li>View the files;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>View the history;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Create and view different branches (variants) of the code;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Management tools such as Issues logs (GitHub) or Kambam board (Dev Ops)</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Create pull requests, review code and get feedback from collaborators;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Perform automated QA and deployment of code (e.g.&nbsp;sending a dashboard live).</li>
 </ul>
+</div>
+</section>
+
+<section id="and-github-and-dev-ops-1" class="title-slide slide level1 center">
+<h1>And GitHub and Dev Ops…</h1>
 <p>GitHub and Dev Ops are generally not a place to:</p>
+<div class="fragment fade-up">
 <ul>
 <li>Run your code;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Edit your code (although there is some basic text editing functionality you can use).</li>
 </ul>
+</div>
+</section>
+
+<section id="what-are-the-differences-between-github-and-dev-ops" class="title-slide slide level1 center">
+<h1>What are the differences between GitHub and Dev Ops?</h1>
+<div class="fragment fade-up">
+<ul>
+<li>GitHub
+<ul>
+<li>Public</li>
+<li>Can publish (Shiny dashboards, quarto and Rmarkdown html) to ShinyApps.io, GitHub pages etc.</li>
+</ul></li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
+<li>Azure DevOps
+<ul>
+<li>Internal to DfE</li>
+<li>Can manage secure permissions</li>
+<li>Can publish (Shiny dashboards, quarto and Rmarkdown html) to rsconnect</li>
+</ul></li>
+</ul>
+</div>
 </section>
 
 <section id="what-are-the-key-benefits-of-using-git" class="title-slide slide level1 center">
 <h1>What are the key benefits of using git?</h1>
+<div class="fragment fade-up">
 <ul>
 <li>Collaboration
 <ul>
 <li>Get support from team and wider community by easily sharing code (our team find it much easier to support you if we can just go and clone your repo…)</li>
 </ul></li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Code QA via reviews and automated checks</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Continuity</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Can undo changes</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Can try different methodologies in a properly controlled way</li>
 </ul>
+</div>
 </section>
 
 <section id="what-does-interacting-with-git-look-like" class="title-slide slide level1 center">
```

Comment:
Is there an argument to add git stash to the list maybe? I don't know how much other people use it but we've definitely used it in the workshops before

Reply:
I think I'm going to leave it out of this one, but have it on the list of things to add for the intermediate git workshop.